cli/dinghy/dnsmasq.rb

```diff
@@ -5,12 +5,18 @@
 class Dnsmasq
   include Dinghy::Daemon
   RESOLVER_DIR = Pathname("/etc/resolver")
-  RESOLVER_FILE = RESOLVER_DIR.join("docker")
 
-  attr_reader :machine
+  attr_reader :machine, :resolver_file, :dinghy_domain
 
   def initialize(machine)
     @machine = machine
+    self.dinghy_domain = "docker"
+    @resolver_file = RESOLVER_DIR.join(@dinghy_domain)
```

Comment:
Yeah I just now picked up on the fact that we're using the setter for `dinghy_domain` instead of the ivar.